docs/api/create_key.md

```diff
@@ -32,9 +32,9 @@ The following describes dummy API requests that return a predictable response. P
 
 Request -> x_client_device_id | Response -> x_status | Response -> x_code
 -----------|-----------|-----------
-1 | Success | SCRK01
-2 | Error | EVAL01
-3 | Error | EAUT01
+01###### | Success | SCRK01
+30###### | Error | EVAL01
+31###### | Error | EAUT01
 any other value | Error | EISE01
 
```

Comment:
Can you also add - # signifies an alphanumeric digit
as we do for Process Authorisation